app/src/main/java/com/padc/interactive_training/adapters/TodoAdapter.java

```diff
@@ -0,0 +1,131 @@
+package com.padc.interactive_training.adapters;
+
+import android.content.Context;
+import android.support.v7.widget.RecyclerView;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.LinearLayout;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import com.daimajia.swipe.SimpleSwipeListener;
+import com.daimajia.swipe.SwipeLayout;
+import com.daimajia.swipe.adapters.RecyclerSwipeAdapter;
+import com.daimajia.swipe.implments.SwipeItemRecyclerMangerImpl;
+import com.padc.interactive_training.InteractiveTrainingApp;
+import com.padc.interactive_training.R;
+
+import java.util.ArrayList;
+
+/**
+ * Created by NayLinAung on 9/13/2016.
+ */
+public class TodoAdapter extends RecyclerSwipeAdapter<TodoAdapter.SimpleViewHolder> {
+
+    public static class SimpleViewHolder extends RecyclerView.ViewHolder {
```

Comment:
You should separate ViewHolder in different Package, considering there's a lot of layout code. Otherwise, your adapter will have many lines of code and will be hard to read